mkdocs.yml

```diff
@@ -4,25 +4,31 @@ site_url: https://www.python-httpx.org/
 
 theme:
     name: 'material'
+    features:
+        - navigation.sections
```

Comment:
* What change does this feature effect?
* Where would I find this documented?

Reply:
This allows showing navigation sections as always-unfolded, instead of foldable groupings as is the default in mkdocs.

This is here: https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-sections

Reply:
Is this clarification enough or are there any changes to the code you'd think worthy?